src/App.js

```diff
@@ -1,30 +1,34 @@
 import React from "react";
+import { Provider } from 'react-redux'
+import store from './redux/store'
 import { Route, Routes } from "react-router-dom"
 import Header from "./Header/Header";
-import Footer from "./components/Footer/Footer";
+import Footer from "./Components/Footer/Footer";
```

Comment:
the name of the componets folder shoulb be with small letter. rename it in your project